### PR TITLE
Add 1.23.1 and 1.23.0 to news section

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

This PR removed old news and updates it with new release notes.

I neglected to do this in sync with the actual release(s), which was an oversight on my part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,33 +215,70 @@ metrics, but also troubleshoot complex performance problems.
...
+Release v1.23.1 of the Netdata Agent is a patch for two significant issues.
 
-Netdata Cloud builds on top of our open source monitoring Agent to give you real-time visibility for your entire infrastructure. Once you've connected your Agents to Cloud, you can view key metrics, insightful charts, and active alarms from all your nodes in a single web interface. When an anomaly strikes, seamlessly navigate to any node to troubleshoot and discover the root cause with the familiar Netdata dashboard.
+PR [#9436](https://github.com/netdata/netdata/pull/9436) fixed an issue where dimensions were marked obsolete and
+archived simultaneously, which caused segmentation faults. We're grateful to [marioem](https://github.com/marioem), who
+first reported the issue, and other members of the Netdata community who contributed their insights and valuable log
+information, which we used to diagnose and fix the bug.
 
-![Animated GIF of Netdata Cloud](https://user-images.githubusercontent.com/1153921/80828986-1ebb3b00-8b9b-11ea-957f-2c8d0d009e44.gif)
+PR [#9428] fixed a significant issue with duplicate alarm IDs, which caused issues in how alarms were sent and displayed
```
Comment (cakrit) [inline README.md:227]: The PR number is missing the link.

Comment (joelhans) [reply README.md]: Fixed

**Comment by joelhans** [comment]
Sorry for the re-requested reviews... I figured I might as well add 1.23.2 now that it's out.

**Comment by aabatangle** [comment]
@cakrit @amoss

**Pull request merged**